src/test/java/io/pravega/connectors/flink/FlinkPravegaTableITCase.java

```diff
@@ -265,11 +280,15 @@ private void testTableSourceStreamingDescriptor(Stream stream, PravegaConfig pra
                 .createStreamTableSource(propertiesMap);
 
         tableEnv.registerTableSource("MyTableRow", source);
-        String sqlQuery = "SELECT user, count(uri) from MyTableRow GROUP BY user";
+        String sqlQuery = "SELECT user, " +
```

Comment:
Switched the test SQL in batch and streaming because the batch mode does not support watermark and has some remaining issues on timestamps.